collection-scripts/gather_service_logs

```diff
@@ -5,7 +5,8 @@ BASE_COLLECTION_PATH="/must-gather"
 ROLES=${1:-master}         ### Defaults to only collecting things from Masters
 
 # Service Lists
-GENERAL_SERVICES=(kubelet crio machine-config-daemon-firstboot machine-config-daemon-host)
+GENERAL_SERVICES=(crio kubelet machine-config-daemon-firstboot machine-config-daemon-host openvswitch ovs-configuration
```

Comment:
could we also add NetworkManager to this list? With ovs-configuration failures, we usually need to also see the NM logs to debug.

Reply:
@trozet it's there now. tested locally and it works.